components/dashboard/src/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { getPaginationNumbers } from "./getPagination";
+import Arrow from "../components/Arrow";
+
+interface PaginationProps {
+    totalResults: number;
+    totalNumberOfPages: number;
+    currentPage: number;
+    setCurrentPage: any;
```

Comment:
nit: I think you can give this a proper type; hovering over a setter should give you the type.

Reply:
Lol totally forgot about this one. Done!

Reply:
Btw it's also on hold because I still need to change the `resultsPerPage` that I changed for this PR.

Reply:
> yeah, I linked your comment in the review summary comment.

My brain is so not working right now 🤣